packages/platforms/accton/x86-64/wedge100bf-32x/onlp/builds/x86_64_accton_wedge100bf_32x/module/src/thermali.c

```diff
@@ -27,13 +27,6 @@
 #include <onlp/platformi/thermali.h>
 #include "platform_lib.h"
 
-#define VALIDATE(_id)                           \
-    do {                                        \
-        if(!ONLP_OID_IS_THERMAL(_id)) {         \
-            return ONLP_STATUS_E_INVALID;       \
-        }                                       \
-    } while(0)
-
 #define THERMAL_PATH_FORMAT "/sys/bus/i2c/drivers/lm75/%s/temp1_input"
 #define THERMAL_CPU_CORE_PATH_FORMAT "/sys/bus/i2c/drivers/com_e_driver/%s/temp2_input"

```

Comment:
I am using OpenBmc version `Open BMC Release 17f7b9a-dirty` and the path is a bit different:
`/sys/bus/i2c/drivers/com_e_driver/4-0033/temp1_input`

Any though?

Reply:
I am using Open BMC version 1.0.0.14 and it uses _/sys/bus/i2c/drivers/com_e_driver/4-0033/temp2_input_ for CPU_CORE temperature and after testing,  I used "temp2_input" instead of "temp1_input".

Reply:
I see, I guess this is OpenBmc problem, not ONL

Reply:
Please merge my commits, if all issues are resolved.